contracts/limitOrder/LimitOrderBook.sol

```diff
@@ -112,13 +113,21 @@ contract LimitOrderBook is
         if (order.reduceOnly) {
             // LOB_ROINS: ReduceOnly Is Not Satisfied
             require((oldTakerPositionSize != 0) && (oldTakerPositionSize < 0 != order.isBaseToQuote), "LOB_ROINS");
-            // if trader has no position, he/she will get reverted
-            // if trader has short position, he/she can only open a long position
+
+            // if trader has no position, order will get reverted
+            // if trader has short position, trader can only open a long position
             // => oldTakerPositionSize < 0 != order.isBaseToQuote => true != false
-            // if trader has long position, he/she can only open a short position
+            // if trader has long position, trader can only open a short position
             // => oldTakerPositionSize < 0 != order.isBaseToQuote => false != true
         }
 
+        if (
```

Comment:
Perhaps we can merge as `order.orderType != ILimitOrderBook.OrderType.LimitOrder`; or is there any concern?

Reply:
No, no concern. Though I prefer explicit if conditions, but no strong opinion on this. Will fix.

Reply:
Related: https://github.com/perpetual-protocol/perp-curie-periphery-contract/pull/14/files#r887582410